src/ledger/CheckpointRange.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include <cstdint>
+
+namespace stellar
+{
+
+class CheckpointRange final
```

Comment:
`CheckpointRange` doesn't seem like a good name, it looks like this is just a `LedgerRange`

Reply:
LedgerRange was one of the names that I considered for that. But LedgerRange suggest that values like LedgerRange{1, 4} are valid, but usage of CheckpointRange requires it to be like CheckpointRange{63, 63} or CheckpointRange{63, 127} (must start and end at checkpoint boundary).

I was not able to assert() that, because checkpoint frequency are configurable.

Reply:
Oh I see, if you were passing a frequency to the constructor, you could check for that there.